package.json

```diff
@@ -51,8 +47,6 @@
     "eslint-plugin-react": "^7.24.0",
     "eslint-plugin-react-hooks": "^5.0.0",
     "eslint-plugin-testing-library": "^7.0.0",
-    "flow-copy-source": "^2.0.9",
```

Comment:
I think `cpy-cli` could be removed as well. This library is often a source of CVEs

Reply:
Thanks, removed it!